Designpage-Script.js

```diff
@@ -34,22 +34,11 @@ document.getElementById("add").onclick = function(){
 }
 
 
+function displayImgChange(imgId){
 
-function displayImg1Change(){
-    document.getElementById("DisplayImg").src = "images/image-product-1.jpg";
+    document.getElementById("DisplayImg").src = document.getElementById(imgId).src;
```

Comment:
The "document.getElementById(imgId)src" is calling the image src of the image id that was click right?

what i don't get is isnt the src of those image with id(img1, img2, etc...) going to be calling the thumbnail and not the actual full image?